exporter/awsxrayexporter/internal/translator/segment.go

```diff
@@ -230,6 +237,34 @@ func MakeSegment(span ptrace.Span, resource pcommon.Resource, indexedAttrs []str
 	}, nil
 }
 
+// fixK8sSegmentName appends EKS/K8s cluster name and namespace to segment name prefix
+// and use '::' as delimiter for the concatenation
+func fixK8sSegmentName(name string, attributes pcommon.Map) string {
+	if namespace, ok := attributes.Get(awsK8sNamespace); ok {
+		name = fmt.Sprintf("%v::%v", namespace.Str(), name)
+	} else {
+		name = fmt.Sprintf("::%v", name)
```

Comment:
Suggestion: I feel instead of leaving the namespace and cluster blank when absent, using `Unknown` might provide a better customer experience. Something like `Unknown::MyNamespace::MySpanName` would look better than `::MyNameSpace::MySpanName`.

Reply:
I think this is way/requirement aligned with the business team.